plugins/rbac-backend/src/service/policies-rest-api.ts

```diff
@@ -52,21 +53,20 @@ export class PolicesServer {
   ) {}
 
   private async authorize(
-    identity: IdentityApi,
     request: Request,
-    permissionEvaluator: PermissionEvaluator,
     permission: QueryPermissionRequest,
-  ) {
-    const user = await identity.getIdentity({ request });
-    if (!user) {
-      throw new NotAllowedError();
+  ): Promise<PolicyDecision> {
+    if (permission.permission !== policyEntityReadPermission) {
```

Comment:
This only works for machine-to-machine tokens now. What would happen if I were a user with read or write permission? We still need to allow users with correct test of rights.

Reply:
During talk in the slack with Gorkem we checked and confirmed, that everything should work fine for user token too.